templates/helm/command-center/templates/argo.yaml

```diff
@@ -66,10 +80,15 @@ spec:
   targetNamespace: argo-ui
   resetValues: true
   chart:
-    repository: https://broadinstitute.github.io/monster-helm
-    name: argo-server
-    version: 0.4.1
+    git: git://github.com/broadinstitute/monster-helm.git
+    ref: dm-argo-27-rc2-DSPDC-926
```

Comment:
does that note to self you have above apply to this too?

Reply:
Ye, updated now